src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
```

Comment:
You should describe what is "user-friendly" to you. Maybe just point to some already existing example. For some people it is "user-friendly" to be able to add definitions to an initialisation file, this is already possible right now.

Reply:
I was thinking about the shortcut editor of GIMP, Inkscape, or many other software. For example, ![XFCE Shortcuts editor](https://docs.xfce.org/_media/xfce/xfce4-settings/xfce4-settings-keyboard-shortcuts.png)

Reply:
Then add the image to the article, this makes clear your idea. But again: I think the forum is the right way to discuss this. Because it could be that TeXmacs do not fit in the same category as many other editors. For example shortcuts in TeXmacs depend on the position of the focus, and this is not possible to represent in a dialog like the one you set up there.